tests/framework/installer/ceph_installer.go

```diff
@@ -753,13 +753,22 @@ spec:
                       #
                       udevadm trigger || true
                       vgimport -a || true
+                      TMP=$(pvs --noheadings --separator=',' -o pv_name,vg_name)
+                      PVS=
+                      for line in $TMP ; do
+                        if [[ $line =~ ,ceph- ]]; then
+                          PVS="$PVS $(echo $line | cut -d, -f 1)"
```

Comment:
Do not use `cut`. Always try to avoid using non-bash builtins when possible, because we never know when a container image might cut out tools like this to trim the image size. You can use bash's built-in string manipulation for what you need here. I believe `${string%substring}` is what you want here.

Reply:
OK, fixed.